doc/manual/manual.tex

```diff
@@ -2045,18 +2053,26 @@ \subsection{Installing and running \aspect{} on Mac OS X}
 \end{verbatim}}
 \dealii{} app opens a terminal window and displays a \dealii{} message.   \dealii{}.app will install all required libraries for \aspect{} (p4est, parMeTiS, and Trilinos) and will include the environment variables needed to use these libraries.
 
+To ensure the correct compilers are picked up when configuring \aspect{}, add the following lines to your .profile or .bash\_profile:
+
+\begin{verbatim}
+export OMPI_CXX=clang++
+export OMPI_CC=clang
+\end{verbatim}
+
 \item Download  the  \aspect{} source from the git repository.
 
 {\footnotesize \begin{verbatim}
 cd $HOME/src
 git clone https://github.com/geodynamics/aspect.git
 \end{verbatim}}
 
-
-Note: you {\bf MUST} \underline{build} and \underline{run} \aspect{} in the terminal window started by \dealii{}.
+If you wish to use Xcode as your IDE it is recommended to skip to step 7 below. Otherwise, proceed with steps 5 and 6 below.
 
 \item Build \aspect{}
 
+Note: you {\bf MUST} \underline{build} and \underline{run} \aspect{} in the terminal window started by \dealii{}.
+
```

Comment:
What does this comment refer to? That you can't run it in Xcode? Maybe that would be worth stating explicitly.